.github/workflows/backend.yml

```diff
@@ -123,7 +123,7 @@ jobs:
           SLS_DEBUG: '*'
 
       - name: Deploy worker
-        run: npm run deploy-worker-staging
+        run: npm run deploy-worker.sh-staging
```

Comment:
Changes to this file shouldn't be necessary. 

deploy-worker is an alias defined in backend/package.json so shouldn't have .sh in it. Same with deploy-worker-integration